fe/fe-core/src/main/java/com/starrocks/catalog/system/information/StreamLoadsSystemTable.java

```diff
@@ -55,6 +55,7 @@ public static SystemTable create() {
                         .column("CHANNEL_STATE", ScalarType.createVarchar(NAME_CHAR_LEN))
                         .column("TYPE", ScalarType.createVarchar(NAME_CHAR_LEN))
                         .column("TRACKING_SQL", ScalarType.createVarchar(NAME_CHAR_LEN))
+                        .column("WAREHOUSE", ScalarType.createVarchar(NAME_CHAR_LEN))
```

Comment:
‌‌I suggest that we should maintain consistency with the previous logic, and add the display of this column when the cluster is running in shared-data mode.